functions/src/emails.ts

```diff
@@ -0,0 +1,85 @@
+import { sendgrid } from "./apis";
+import Lob from "./lob";
+
+export function notifyUserOnLetterCreate({
+  to,
+  lobResponses,
+}: {
+  to: string;
+  lobResponses: Lob.CreateResponse[];
+}): Promise<void> {
+  const responseSummaries = lobResponses
+    .map((lobResponse: any) => {
+      return `Sent to: ${lobResponse.to.name}
+Expected Delivery: ${lobResponse.expected_delivery_date}
+Preview: ${lobResponse.url}`;
```

Comment:
im not sure what the common practice is for template strings in js/ts, but would it make since to declare these (and others) as a constant for readability?

Reply:
it doesn't work as a constant since it's using string interpolation.

it has to have this awkward alignment because javascript doesn't have a multi-line alignment thing, so if we nicely align the text, we get all the spacing at the start of the line which we don't want.

I thought about using a templating library for this but ... it really seems like overkill